source/containers.tex

```diff
@@ -12953,6 +13017,155 @@
 \end{codeblock}
 \end{itemdescr}
 
+\rSec1[assoc.format]{Associative formatting}
+
+\pnum
+For each of
+\tcode{map},
+\tcode{multimap},
+\tcode{unordered_map}, and
+\tcode{unordered_multimap},
+the library provides the following formatter specialization
+where \placeholder{map-type} is the name of the template:
```

Comment:
I think this should be inside `tcode` so that _map-type_ is formatted in code font, consistent with its appearances in the `formatter<map-type<Key, T, U...>, charT>` specialiation.

Reply:
fixed